src/views/NoteListView.cpp

```diff
@@ -485,6 +494,18 @@ void NoteListView::onShowNoteInfoAction()
     Q_EMIT noteInfoDialogRequested(noteLocalUid);
 }
 
+
+void NoteListView::onToggleThumbnailPreference() {
+    QNDEBUG(QStringLiteral("NoteListView::onToggleThumbnailPreference noteLocalUid="));
```

Comment:
First, ` noteLocalUid=` seems redundant here. Second, please insert an empty line after this `QNDEBUG` statement.